yt/frontends/ytdata/data_structures.py

```diff
@@ -86,7 +86,7 @@ def _parse_parameter_file(self):
             for key in f.attrs.keys():
                 v = parse_h5_attr(f, key)
                 if key == "con_args":
-                    v = v.astype("str")
+                    v = eval(v)
```

Comment:
This is the pair to the `repr()` call above, I think.  I'm not totally sure this is safe.  What's the use case for this particular set of changes?

Reply:
I explained above in the question about `repr`. Not sure what you mean by "safe" - are you talking about a security concern or something? This is all data that's internal to yt...